static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -533,64 +539,100 @@ require.def(
             },
 
             _getClampedTimeForPlayFrom: function (seconds) {
+                if (this._currentPlayer === this.PlayerEmps.StreamingPlayer && !this._updatingTime) {
```

Comment:
As noted elsewhere, we should just update the seekable range on an 8 second interval, so this wouldn't be necessary.